src/DynamicData.Tests/Cache/EditDiffChangeSetOptionalFixture.cs

```diff
@@ -190,7 +190,7 @@ private class PersonComparer : IEqualityComparer<Person>
     {
         public bool Equals([DisallowNull] Person x, [DisallowNull] Person y) =>
             EqualityComparer<string>.Default.Equals(x.Name, y.Name) && EqualityComparer<int>.Default.Equals(x.Id, y.Id);
-        public int GetHashCode([DisallowNull] Person obj) => throw new NotImplementedException();
+        public int GetHashCode([DisallowNull] Person obj) => obj.GetHashCode();
```

Comment:
I'm guessing this (and the others) started failing some test with .NET 9?

Reply:
Yes, I didn't see any test's specifically looking for the not implemented exception for the GetHashcode ones, so figured implementing was the better option